### ISSUE TITLE
[frameit] Wrong background image is used for Screenshot generation

### ISSUE THREAD

**Issue by patreu22** [fastlane/fastlane#17225]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I'm using the frameit feature of Fastlane to generate my screenshots, frame them into a device and to put the device on a single colored background and add some text. Eventually it's supposed to look like this:

![iPad Pro (12 9-inch) (3rd generation)-1_framed](https://user-images.githubusercontent.com/11362357/92977974-bce6d800-f48e-11ea-93bd-368cbde23c55.png)

I'm taking 4 screenshots in total, each one is supposed to be placed on a different color. Instead the first 3 screenshots are placed on the yellow background and the 4th image is placed on the brown background. This doesn't apply to iPhone screenshot, on these everything just works, but I'm struggeling on iPads. On different iPads different backgrounds are chosen, but not the correct ones. The only indication is given by the Magick warnings, but I'm not sure how to debug these.

As background images I use these 4:

![brown_background](https://user-images.githubusercontent.com/11362357/92977986-c708d680-f48e-11ea-8da1-ce9c6f96aad1.jpg)
![green_background](https://user-images.githubusercontent.com/11362357/92977987-c708d680-f48e-11ea-94fa-aad87cc41e8f.jpg)
![red_background](https://user-images.githubusercontent.com/11362357/92977988-c7a16d00-f48e-11ea-8c7b-a9f8600a9bdd.jpg)
![yellow_background](https://user-images.githubusercontent.com/11362357/92977990-c7a16d00-f48e-11ea-8bb7-7623136150a5.jpg)

##### Command executed

`frame_screenshots(white: false)` with the following `framefile.json`:
```
{
    "default": {
        "title": {
            "fonts": [
                {
                    "font": "./fonts/Roboto.ttf",
                    "supported": [
                        "de-DE",
                        "en-US",
                        "es-ES",
                        "fr-FR",
                        "it-IT",
                        "nl-NL",
                        "sv-SE",
                        "pt-BR"
                    ]
                }
            ],
            "color": "#FFFFFF"
        },
        "padding": 100,
        "show_complete_frame": false,
        "font_scale_factor": 0.25
    },
    "data": [
        {
            "filter": "1",
            "background": "./backgrounds/red_background.jpg",
            "title": {
                "text": "Lerne alle\nPrüfungsfragen"
            }
        },
        {
            "filter": "2",
            "background": "./backgrounds/green_background.jpg",
            "title": {
                "text": "Statistiken helfen\nbeim Lernen"
            }
        },
        {
            "filter": "3",
            "background": "./backgrounds/yellow_background.jpg",
            "title": {
                "text": "Authentische\nFragebögen"
            }
        },
        {
            "filter": "4",
            "background": "./backgrounds/brown_background.jpg",
            "title": {
                "text": "Detaillierte\nAnleitungen"
            }
        }
    ]
}
```

##### Complete output when running fastlane, including the stack trace and command used

```
[00:18:30]: -------------------------------
[00:18:30]: --- Step: frame_screenshots ---
[00:18:30]: -------------------------------
[00:18:30]: Framing screenshots at path ./fastlane/ (via frameit)
[00:18:30]: Using device frames version 'latest'
[⠴] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-4.png' composite: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-1d1wxtm.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[⠙] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-4.png' convert: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-at1cue.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[✔] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-4.png' 
[00:18:45]: Added frame: '/Users/patrick/Desktop/repos/flutter_skeleton/ios/fastlane/screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-4_framed.png'
[⠧] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-2.png' composite: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-1p4cqn6.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[⠹] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-2.png' convert: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-hzzw3g.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[✔] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-2.png' 
[00:19:00]: Added frame: '/Users/patrick/Desktop/repos/flutter_skeleton/ios/fastlane/screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-2_framed.png'
[⠦] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-3.png' composite: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-1qjtq5l.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[⠙] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-3.png' convert: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-k32zn2.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[✔] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-3.png' 
[00:19:15]: Added frame: '/Users/patrick/Desktop/repos/flutter_skeleton/ios/fastlane/screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-3_framed.png'
[⠧] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-1.png' composite: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-j7mbje.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[⠹] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-1.png' convert: invalid profile length `/var/folders/5h/6hz8bq9d6szdtssgyhztczl40000gn/T/mini_magick20200912-89268-op1icw.png' @ warning/png.c/MagickPNGWarningHandler/1748.
[✔] Framing screenshot './screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-1.png' 
[00:19:29]: Added frame: '/Users/patrick/Desktop/repos/flutter_skeleton/ios/fastlane/screenshots/de-DE/iPad Pro (12.9-inch) (3rd generation)-1_framed.png'
[✔] Framing screenshot './iap-screenshot.png' 
[00:19:33]: Added frame: '/Users/patrick/Desktop/repos/flutter_skeleton/ios/fastlane/iap-screenshot_framed.png'
```
### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.4                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | true                                                           |
| Git                         | git version 2.28.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.4 (19E287)                                      |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.6                                                           |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | de_DE.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| bundler                   | 2.0.2        |
| rake                      | 13.0.1       |
| CFPropertyList            | 3.0.2        |
| concurrent-ruby           | 1.1.5        |
| i18n                      | 0.9.5        |
| minitest                  | 5.13.0       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.5        |
| activesupport             | 4.2.11.1     |
| public_suffix             | 4.0.6        |
| addressable               | 2.7.0        |
| httpclient                | 2.8.3        |
| json                      | 2.3.1        |
| algoliasearch             | 1.27.1       |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.367.0      |
| aws-sigv4                 | 1.2.2        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.105.0      |
| aws-sdk-kms               | 1.37.0       |
| aws-sdk-s3                | 1.80.0       |
| babosa                    | 1.0.3        |
| claide                    | 1.0.3        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| netrc                     | 0.11.0       |
| ffi                       | 1.12.2       |
| ethon                     | 0.12.0       |
| typhoeus                  | 1.3.1        |
| cocoapods-core            | 1.9.1        |
| cocoapods-deintegrate     | 1.0.4        |
| cocoapods-downloader      | 1.3.0        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.1.0        |
| cocoapods-trunk           | 1.4.1        |
| cocoapods-try             | 1.1.0        |
| colored2                  | 3.1.2        |
| escape                    | 0.0.4        |
| fourflusher               | 2.3.1        |
| gh_inspector              | 1.1.3        |
| molinillo                 | 0.6.6        |
| ruby-macho                | 1.4.0        |
| nanaimo                   | 0.3.0        |
| xcodeproj                 | 1.18.0       |
| cocoapods                 | 1.9.1        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| digest-crc                | 0.6.1        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| dotenv                    | 2.7.6        |
| emoji_regex               | 3.0.0        |
| excon                     | 0.76.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.7        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.2.0        |
| jwt                       | 2.2.2        |
| memoist                   | 0.16.2       |
| multi_json                | 1.15.0       |
| os                        | 1.1.1        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.1       |
| mini_mime                 | 1.0.2        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.3        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.28.0       |
| mini_magick               | 4.10.1       |
| plist                     | 3.5.0        |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.8.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| xcpretty-travis-formatter | 1.0.0        |
</details>
```

**Comment by patreu22** [comment]
Just to give a better example, this is how the latest screenshots look for the iPhones:
![image](https://user-images.githubusercontent.com/11362357/92999423-fd8f3180-f520-11ea-8478-91f47c2a3da0.png)

That's how they look for the iPad Pro (3rd generation):
![image](https://user-images.githubusercontent.com/11362357/92999434-18fa3c80-f521-11ea-980d-7ee2a84d12e6.png)


And that's how they look for the iPad Pro (2nd generation):
![image](https://user-images.githubusercontent.com/11362357/92999442-28798580-f521-11ea-93e9-89cbcd32342b.png)



All of them were generated in the same run of frameit, I'm really confused.

**Comment by patreu22** [comment]
Any chance somebody might look into that?

**Comment by Johnson2234** [comment]
is there any one who can give solution of it please suggest any solution [MCDVOICE](https://www.mcdvoice.us/)

**Comment by patreu22** [comment]
Do you have the same problem @Johnson2234? Maybe @joshdholtz knows who has an eye on frameit?

**Comment by patreu22** [comment]
Hey there, little follow-up for @joshdholtz after this issue still being a thing over 2 month later. Any idea who could be the right person to contact about this?

**Comment by joshdholtz** [comment]
Thanks for ping! Will move this to the top of my list for Monday morning

**Comment by patreu22** [comment]
Cheers, enjoy the rest of the weekend 🍻

**Comment by patreu22** [comment]
Hey @joshdholtz - I know you just became a dad (congrats on that) but a couple of Mondays passed and the Bot is forcing me to ask again, cause I would like to avoid this issue being closed 😁

**Comment by joshdholtz** [comment]
@patreu22 Fixed! Thanks for ping ❤️

**Comment by patreu22** [comment]
Awesome! Does it mean it will be fixed in the next release? Would be a relieve for our marketing guys :D

**Comment by patreu22** [comment]
Hey @joshdholtz, could you provide any reference to the PR or did the feature get shipped at all? I just ran fastlane 2.172.0 and the problem didn't seem to be fixed

**Comment by patreu22** [comment]
> Hey @joshdholtz, could you provide any reference to the PR or did the feature get shipped at all? I just ran fastlane 2.172.0 and the problem didn't seem to be fixed

Hey @joshdholtz - I couldn't track the fix so far in any release, maybe you could elaborate on this and we can close this issue?
